SamplesDashboard/SamplesDashboard/Services/AzureSdkService.cs

```diff
@@ -82,13 +82,17 @@ public async Task<string> GetAzureSdkVersions(string packageName)
             string azureSdkVersion = String.Empty;
 
             var sdks = await FetchAzureSdkVersions();
-            foreach (var sdk in sdks)
+            if (sdks != null)
```

Comment:
you can reduce the nesting by saying 

if (sdks == null){
return String.Empty
}